cmd/main.go

```diff
@@ -273,6 +274,9 @@ func main() {
 			"Failed to parse RELEASE_SOURCES json %s", Options.ReleaseSourcesConfig.ReleaseSources)
 	}
 
+	var ignoredOpenshiftVersions = []string{}
+	parseIgnoredOpenshiftVersions(ignoredOpenshiftVersions, failOnError)
+
 	log.Println(fmt.Sprintf("Started service with OS images %v, Release images %v",
```

Comment:
Consider logging also the IgnoredOpenshiftVersions, could be useful for debugging,